graphql/server.go

```diff
@@ -18,8 +18,9 @@ import (
 )
 
 const (
-	MaxSubscriptions = 200
-	MinRerunInterval = 5 * time.Second
+	MaxSubscriptions       = 200
+	MaxConcurrentRerunners = 64
```

Comment:
Could we make this configurable on the server? I regret these constants and would like to not add any more.